src/Libraries/CoreNodeModelsWpf/Converters/StringToDateTimeOffsetConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace Dynamo.Wpf.Converters
+{
+    public class StringToDateTimeOffsetConverter : IValueConverter
```

Comment:
This converter converts a date/time string into a `DateTime` object. If conversion fails, it returns a new `DateTime` object.